public/js/graph/react_graph.js

```diff
@@ -150,10 +150,21 @@ var Graph = React.createClass({
             zoomFactor: 1,
             horizontalPanFactor: 0,
             verticalPanFactor: 0,
-            mouseDown: false
+            mouseDown: false,
+            graphMode: 'graph',
```

Comment:
You can use a prop to set the initial value rather than doing what you did below. See https://facebook.github.io/react/tips/props-in-getInitialState-as-anti-pattern.html for details.